kintoclient/__init__.py

```diff
@@ -0,0 +1,254 @@
+import requests
+import urlparse
+import json
+import six
+import uuid
+
+DEFAULT_SERVER_URL = 'https://kinto.dev.mozaws.net/v1'
+
+# XXX rename to 'objects'?
```

Comment:
+1 to rename to objects (see http://kinto.readthedocs.org/en/latest/permissions.html)